code/modules/events/event_dynamic.dm

```diff
@@ -99,6 +99,8 @@ var/list/event_last_fired = list()
 	//and start working via the constructor.
 	new picked_event
 
+	score["eventsendured"]++
```

Comment:
Yes but you put events ENDURED, hardly an accurate statement to say they 'endured' pda spam